lib/src/validators/collection_validators.dart

```diff
@@ -0,0 +1,311 @@
+import '../../localization/l10n.dart';
+import 'constants.dart';
+
+/// {@template validator_min_length}
+/// Creates a validator function that checks if the input's length is greater
+/// than or equal to `min`. The validator returns `null` for valid input and an
+/// error message for invalid input.
+///
+/// If validation fails and no custom error message generator is provided via
+/// [minLengthMsg], returns the default localized error message from
+/// `FormBuilderLocalizations.current.minLengthErrorText(min)`.
+///
+/// ## Type Parameters
+/// - `T`: The type of input to validate. Must extend `Object` and can be a
+///   `String`, `Iterable`, `Map`, or other object type.
```

Comment:
Could be another object type? Make sense this?
If validator recieve for example an int, always would be valid because `valueLength = 1` and `min =< 1`

Reply:
I think that allow only collections is the best way

Reply:
Now, the collection validators only accept collections.